examples/ai-lab-example/storybook/main.js

```diff
@@ -11,5 +11,11 @@ module.exports = {
         scriptsPackageName: 'react-scripts'
       }
     }
-  ]
+  ],
+  webpackFinal: async (config) => {
+    // We are an example and therefore we want to use the source because we can!
+    config.resolve.alias['ai-lab'] = path.resolve(__dirname, '../../../packages/ai-lab/src');
```

Comment:
This is the actual change. Basically we are forcing Webpack to resolve to the source of the ai-lab code instead of using the compiled code. This reduces the cycle time on seeing changes occur in the example fairly significantly.